etc/adminhtml/system.xml

```diff
@@ -9,6 +9,10 @@
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                     <comment>Warning: Enabling 2FA will immediately prompt admin user for OTP code.</comment>
                 </field>
+                <field id="enable_api" translate="label" type="select" sortOrder="100" showInDefault="1" canRestore="1">
+                    <label>Enable 2FA for token service</label>
```

Comment:
After further thinking, since this just enables the token generation, the label should probably say "Enable 2FA for API token generation", and the config path probably be updated similarly:

`twofactorauth/general/enable_for_api_token_generation`

The README also needs to be updated on what this code is doing and how to set it.